### ISSUE TITLE
Donation Block: Can't navigate via keyboard

### ISSUE THREAD

**Issue by laurelfulford** [Automattic/newspack-blocks#123]
When navigating through the page using a keyboard, you can't operate the donation block:

* The tabs don't get focus so you can't switch them
* The dollar values don't get focus so you can't switch them:

The only values that you can focus on by tabbing is the 'Donate' button, and, if already opened, the 'Other' text field.

![image](https://user-images.githubusercontent.com/177561/65377558-74715180-dc62-11e9-804c-2148eae0eee2.png)

It should be possible to change all of the settings using just a keyboard, and to get visual feedback on where you current are.
